blocks/components/editable/index.js

```diff
@@ -91,3 +115,7 @@ export default class Editable extends wp.element.Component {
 		);
 	}
 }
+
+Editable.defaultProps = {
```

Comment:
Aside: I'm wary of the future of `defaultProps`, especially since in most cases it can be achieved by destructured object defaults. We're doing this already for the component with the `tagName` prop (see `render`) so we should at least choose a consistent approach.